dali/test/python/test_RN_external_source_parallel.py

```diff
@@ -0,0 +1,291 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
I think you need to add this test somewhere to be called.
Maybe TL1_jupyter_plugins/test_pytorch.sh for training and TL0_python-self-test/test_body.sh, TL0_python-self-test_xavier/test_body.sh and TL1_python-self-test_conda/test_body.sh, 
Also make sure that
```
import torch
import torch.nn as nn
import torch.optim
import torchvision.models as models
```
is not called if training_test is not called as well.

Reply:
Moved them into training proc